### PR TITLE
Added 'Archive Events' Page to filter events in the admin console

### PR DESCRIPTION
**Pull request by mscanlan-git**
This PR addresses #2026 by adding an "Archived Events" page to eliminate event clutter that will eventually occur over time as we add new events. 

EDIT: We will eventually need to change the normal events page to only include events that haven't been archived yet (active/future events). This can be done in a separate PR, though.

### CODE REVIEW

File: physionet-django/console/views.py [hunk 1]
```diff
@@ -2952,6 +2952,21 @@ def event(request):
                    })
 
 
+@permission_required('user.view_all_events', raise_exception=True)
+def archive_event(request):
+    """
+    List of archived events
+    """
+    now = timezone.now()
```
Comment (tompollard) [inline physionet-django/console/views.py:2960]: Not sure that it is necessary to assign the `now` variable here. `Event.objects.filter(end_date__lte=timezone.now())` is just as readable.

**Review by tompollard — APPROVED** [review]
This is great @mscanlan-git, nice work! I have listed a few quick thoughts below. As this is a private part of the site I'd be happy to merge as it is, and perhaps these can be addressed in a couple of new pull requests?

- For the URL, instead of http://localhost:8000/console/archive_event/, we could consider switching to something like http://localhost:8000/console/event/archived/. This is consistent with the approach for the user pages in the console (e.g. http://localhost:8000/console/users/inactive/).
- For the menu, instead of "Event > Archived Events", we could do: "Event > Archived Events". We know that we are in the Events submenu, say no need to say this again. We might also want to change "Events > Event list" to something like "Events > Active" for consistency?

**Comment by tompollard** [comment]
Now on the live site, thanks again! BTW, minor thing, but don't forget to clean up commit logs before opening a pull request (e.g in this case rolling back the `temp` commit with `git reset XXX` and then adding a fresh one).

**Comment by mscanlan-git** [comment]
Thanks @tompollard , I will be looking to add those changes you mentioned in my next PR to address the active events page, to only allow currently actively events. 

Also, will keep note on cleaning commits, forgot about the temp commit from yesterday!

**Pull request merged**